sensor/common/scannerclient/grpc_client.go

```diff
@@ -177,7 +197,7 @@ func (c *v2Client) GetImageAnalysis(ctx context.Context, image *storage.Image, c
 		return nil, errors.Wrap(err, "getting image components from scanner")
 	}
 
-	log.Debugf("Received image components from local Scanner for image %s", imgName)
+	log.Debugf("Received image components from local Clairify scanner for image: %q", imgName)
```

Comment:
there is another debug a few lines above this which just says `Scanner`. I'm ok with just `Scanner` since we know we differentiate between Scanner and Scanner v4